### DISCUSSION TITLE
Clock overlapping GIMP tooltips

### DISCUSSION THREAD

**Discussion by snowmobile6** [martinet101/ElevenClock#864] [Q&A] [answered]
![ElevenClock covering GIMP tooltips](https://user-images.githubusercontent.com/110550062/182639714-28c36ddd-43b6-477c-9e4e-c00fc2cf30ea.png)
ElevenClock is covering tooltips in GIMP regardless of whether or not the GIMP window is maximized (full-screen) or not.

Is there a fix for this in the ElevenClock settings that I'm just missing?

**Comment by marticliment** [comment, marked as answer]
No, there's no real fix. 

That happens because of how elevenClock workbs. It shows on top of everything to be able to overlay the taskbar. So, when there's some kind of menu/tooltip, it gets overlapped.

If the issue annoys you too much, you can right-click a clock and then click hide, to hide it temporarily. Then click the system tray icon to show it again
